tap_github/streams.py

```diff
@@ -158,6 +157,40 @@ def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
     ).to_dict()
 
 
+class ReadmeStream(GitHubStream):
+    name = "readme"
+    path = "/repos/{org}/{repo}/readme"
+    primary_keys = ["url"]
```

Comment:
If there should only be one README record result per repo, what about using the primary key of the parent: `["repo", "org"]`

Reply:
Do you know (or can you test) how this will behave if the rep has both `README.md` and also something like `README.rst`. This speaks to the primary key question, I think, whether the API will send both records or pick a winner and send just one.

If just one, I think we want the primary key to reflect the repo itself - if two or more is allowed, I think we would key in based on the `url`.

Wdyt?

Reply:
I did a quick test, and the api just ends up choosing one over the other and only sends one. It sends this if there is no README: 
```
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/repos#get-a-repository-readme"
}
```
I'll change the key to the repo/org combination :+1: